.github/workflows/build.yml

```diff
@@ -4,9 +4,6 @@ on:
   push:
     branches:
       - develop
-  pull_request:
```

Comment:
Remove trigger: pull_request, as the trigger always causes failure for secret problem.